<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HamingTables</class>
 <widget class="QMainWindow" name="HamingTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1449</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HamingTables</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>1061</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>1371</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Numero</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero en Binario</string>
    </property>
   </widget>
   <widget class="QLabel" name="numeroBin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero en Decimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="numeroDec">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-    -    -    -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero en BDC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero en Hexadecimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="primeroH">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="segundoH">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="tercerH">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="BCD2">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>     -</string>
    </property>
   </widget>
   <widget class="QLabel" name="BCD3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>     -</string>
    </property>
   </widget>
   <widget class="QLabel" name="BCD1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>    -</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1449</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
